syncer/syncer.go

```diff
@@ -929,11 +929,12 @@ func (s *Syncer) syncDDL(tctx *tcontext.Context, queueBucket string, db *DBConn,
 			}
 		}
 		if err != nil {
-			s.appendExecErrors(&ExecErrorContext{
-				err:      err,
-				location: sqlJob.currentLocation.Clone(),
-				jobs:     fmt.Sprintf("%v", sqlJob.ddls),
-			})
+			s.jobWg.Done()
```

Comment:
Can you add any explanation about this update?

Reply:
https://github.com/pingcap/dm/pull/1032/commits/914c2852f7e9308993de0b26e8e7984229322d79

hope we could refine error handling soon 😂

Reply:
#1055 fix bug the execError. Agree to refine the error handling later because we now remove the query-error and some code no needed.

Reply:
going to wait #1055 and merge master with that PR into this one